### ISSUE TITLE
what concert data to show on dashboard?

### ISSUE THREAD

**Issue by chrsep** [chrsep/nemesis#7]
1. Audience Gender? So we need to ask users for their gender?
2. Age?
3. Revenue
4. Order count
5. How many users view it
6. How long does the user view the broadcast

**Comment by christianto88** [comment]
Related to #5 @chrsep

**Comment by christianto88** [comment]
@chrsep iya. age n gender biasanya mandatory fields di registration form. dari 2 fields itu bisa muncul banyak dashboard yang bisa ditambahkan , contoh :
- audience's average age/gender di particular concert
- genre yg disukai based on gender/age
